### ISSUE TITLE
Tax formatting for Australia

### ISSUE THREAD

**Issue by CWalkden** [tendenci/tendenci#1136]
Our organisation has grown to a size where we are required to charge tax on the things we sell.  Tax is to be included on memberships and other items that we sell, but not on donations.
In Australia, the tax is called GST (Goods & Services Tax), and is 10%.  Prices are typically quoted including GST, and then the GST is listed on the invoice as a component of the total.

Tendenci makes it a fiddle to get the GST showing correctly.  Here is a screenshot to demonstrate - this is from a form where we sell an item: [https://www.aeva.asn.au/forms/ev-licence-plate-tag/](url)

![image](https://user-images.githubusercontent.com/64681720/201544322-12b346f6-a881-460d-b780-7b60e35448ff.png)

We'd like to list the prices as $5.50 and $9.50, GST inclusive.  But the system only allows the prices to be listed excluding the tax.  This results in our odd-looking number $8.64.  We set this up in the form like so:
![image](https://user-images.githubusercontent.com/64681720/201544943-988509a0-35e8-4092-a3c9-3c9a8c541695.png)

Would it be possible to add an option to display prices including the tax component?

Similarly, in the memberships module:
![image](https://user-images.githubusercontent.com/64681720/201545300-6aed22f0-1ec3-46f5-9202-4d01c86c4465.png)

All prices exclude GST, but we'd prefer to have the membership prices displayed including GST.  The formatting would need to make this clear, here are some suggestions:  

- $55.00 (tax inclusive)
- $55.00 including GST
- $55.00 including $5.00 GST
- $55.00 optional text that we enter

**Comment by goetzk** [comment]
Hi Chris,
Congrats on the growth :)

You'll probably also now encounter the issue in the invoices. We did some crazy stuff in the templates (which I won't share) but would also like the handling of tax to be improved.
IIRC I previously raised the 'Tax rate' dialogue but as your sceenshot shows, I didn't win that battle :)

**Comment by jennyq** [comment]
Hi @CWalkden, 

For the custom forms, you can add the tax info to the "Pricing Description" field to inform user % tax will be applied.

For the memberships application forms, I've added the help text for the membership type field https://github.com/tendenci/tendenci/commit/68400a455d1f3d8af57c6d484e0d6857b17e2426.

I hope that addresses your issue. If not, a pull request is welcome.

@goetzk, I did a search and found this issue you opened years ago https://github.com/tendenci/tendenci/issues/528. Looks like we missed it somehow. Is this the issue you mentioned above?

**Comment by goetzk** [comment]
@jennyq pull request #485 is what I had in mind.

**Comment by CWalkden** [comment]
> For the custom forms, you can add the tax info to the "Pricing Description" field to inform user % tax will be applied.

Yes, but the problem is that this is not the usual way that prices are displayed.  Advertised prices are normally shown with the tax included, so our Tendenci website looks foreign.
I'll look up what a 'pull request' is.

**Comment by goetzk** [comment]
> @goetzk, Sorry but I fully agree with Ed's decision on closing that PR.

I definitely don't want to derail this request discussing the other - I just mentioned it for CW since I know his usecase :)

**Comment by CWalkden** [comment]
> I definitely don't want to derail this request discussing the other - I just mentioned it for CW since I know his usecase :)

Hi @goetzk,
I think your previous issue was suggesting that the 'with tax' value could be entered in from the start.  I assume it would be easier to just change where it is displayed?

@jennyq above suggested a Pull Request.  I've looked into it, and it doesn't look like something within my skillset!
If you're not in a position to work on it, perhaps @rob-hills might like to look at this?

In summary, what we'd like, is:
**Forms:**
In the form design page, add an option to display prices with or without tax included.  This won't affect the existing method of determining prices, just how they are displayed.
**Membership Applications:**
Add an option to display prices with or without tax included.  Add another optional text field to display after the price.  For example the text might be 'including tax'.

The form doesn't need the extra text field, as there is already a description that can be used.

**Comment by CWalkden** [comment]
I just realised that there is another place where prices can be displayed - the Events module.  I assume this would be similar to the Forms module discussed above.

**Comment by rob-hills** [comment]
Hi @jennyq could you have a look at #1413 submitted by @rockinrobstar as these will enable us to close this longstanding issue

**Comment by jennyq** [comment]
That commit has been merged.
